common/src/main/scala/com/gu/threadpools/CustomPool.scala

```diff
@@ -0,0 +1,11 @@
+package com.gu.threadpools
+
+import java.util.concurrent.Executors
+import com.typesafe.scalalogging.StrictLogging
+import scala.concurrent.ExecutionContext
+
+object CustomPool extends StrictLogging {
+
+  implicit val ec = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(4))
```

Comment:
This doesn't need to be implicit, and could have a more meaningful name (`executionContext`)